utils/SlackEventsAPIUtils/chessterUtils.ts

```diff
@@ -0,0 +1,218 @@
+// -----------------------------------------------------------------------------
```

Comment:
Is this actually used aside from the tests? If not, let's remove it.

Reply:
We can put it back in with a PR that causes it to be used, this sort of refactoring and tests is quite a good step towards what we want, but I don't like having unused code in master